packages/icons/package.json

```
@@ -17,38 +17,48 @@
       "types": "./dist/iconsMetaData.d.ts",
       "import": "./dist/iconsMetaData.js",
       "default": "./dist/iconsMetaData.js"
+    },
+    "./raw": {
+      "types": "./dist/svg/index.d.ts",
+      "import": "./dist/svg/index.js",
+      "default": "./dist/svg/index.js"
     }
   },
   "typesVersions": {
     "*": {
       "meta": [
         "./dist/iconsMetaData.d.ts"
+      ],
+      "raw": [
+        "./dist/svg/index.d.ts"
       ]
     }
   },
   "scripts": {
     "plop": "plop",
-    "build": "yarn clean && yarn build:react-icons && rollup -c",
+    "build": "yarn clean && yarn build:react-icons && rollup -c && yarn generate-svg-index",
```

I think the generate should happen before the build by rollup
is it? or it doesn't matter?
```suggestion
    "build": "yarn clean && yarn build:react-icons && yarn generate-svg-index rollup -c",
```

---

oh I think the order matters as it should copy in rollup the index as well, right?

---

I generate it in the dist after it is copied. Would it be better to change it to generate it in the svg folder and then add it to the gitignore and have it copied with the rest of the files?

---

sounds right to me

---

Updated